src/librustc_infer/infer/region_constraints/mod.rs

```diff
@@ -489,7 +489,7 @@ impl<'tcx> RegionConstraintCollector<'_, 'tcx> {
     }
 
     /// Removes all the edges to/from the placeholder regions that are
-    /// in `skols`. This is used after a higher-ranked operation
+    /// in `placeholders`. This is used after a higher-ranked operation
     /// completes to remove all trace of the placeholder regions
     /// created in that time.
     pub fn pop_placeholders(&mut self, placeholders: &FxHashSet<ty::Region<'tcx>>) {
```

Comment:
Hmm, this appears to be dead code since b68fad670bb3612cac26e50751e4fd9150e59977. I guess this and `Purged` can be removed.

Reply:
done